tests/console/commands/test_add.py

```diff
@@ -69,12 +69,45 @@ def tester(command_tester_factory: CommandTesterFactory) -> CommandTester:
     return command_tester_factory("add")
 
 
+@pytest.fixture(autouse=True)
+def repo_add_default_packages(repo: TestRepository) -> None:
+    msgpack_optional_dep = get_dependency("msgpack-python", ">=0.5 <0.6", optional=True)
+    msgpack_dep = get_dependency("msgpack-python", ">=0.5 <0.6")
+    msgpack = get_package("msgpack-python", "0.5.6")
+    repo.add_package(msgpack)
+
+    redis_dep = get_dependency("redis", ">=3.3.6 <4.0.0", optional=True)
+    redis = get_package("redis", "3.4.0")
+    repo.add_package(redis)
+
+    cachy010 = get_package("cachy", "0.1.0")
+    cachy010.extras = {"msgpack": [msgpack_dep]}
+    cachy010.add_dependency(msgpack_optional_dep)
+    repo.add_package(cachy010)
+
+    cachy020 = get_package("cachy", "0.2.0")
+    cachy020.extras = {"msgpack": [get_dependency("msgpack-python")]}
+    cachy020.add_dependency(msgpack_dep)
+    repo.add_package(cachy020)
+
+    cleo065 = get_package("cleo", "0.6.5")
+    cleo065.add_dependency(msgpack_optional_dep)
+    cleo065.add_dependency(redis_dep)
+    cleo065.extras = {
+        canonicalize_name("redis"): [redis_dep],
+        canonicalize_name("msgpack"): [msgpack_dep],
+    }
+    repo.add_package(cleo065)
+
+    repo.add_package(get_package("pendulum", "1.4.4"))
+    repo.add_package(get_package("tomlkit", "0.5.5"))
+    repo.add_package(get_package("pyyaml", "3.13"))
+    repo.add_package(get_package("pyyaml", "4.2b2"))
+
+
 def test_add_no_constraint(
     app: PoetryTestApplication, repo: TestRepository, tester: CommandTester
```

Comment:
I suppose we can remove `repo` from many test functions because we do not use it explicitly anymore?

Reply:
Right. I went through the file and removed all unnecessary `repo` fixture declarations.